app/exporters/section_publishing_api_exporter.rb

```diff
@@ -13,17 +13,13 @@ def initialize(organisation, manual, section, update_type: nil)
   end
 
   def call
-    Services.publishing_api.put_content(content_id, exportable_attributes)
+    Services.publishing_api.put_content(section.uuid, exportable_attributes)
   end
 
 private
 
   attr_reader :organisation, :manual, :section
 
-  def content_id
-    section.uuid
-  end
-
```

Comment:
Minor: This seems a bit orthogonal to the purpose of the PR. Also I'm doing some significant refactoring in this area and so it might make life easier from a merging point of view if this commit was left out.